RaceTrack/Tournament/Tournament.cs

```diff
@@ -0,0 +1,64 @@
+﻿using DataRace;
+using System.Collections.Concurrent;
+
+namespace TournamentManagement
+{
+    public class TournamentResult
```

Comment:
Also there is some inconsistency, because file named `Tournament` and class inside of it - `TournamentResult`